rpcclient/README.md

```diff
@@ -23,15 +23,11 @@ implement and the API is not stable yet.
   Connects to a dcrd RPC server using TLS-secured websockets, registers for
   block connected and block disconnected notifications, and gets the current
   block count
-* [dcrwallet Websockets Example](https://github.com/decred/dcrd/tree/master/rpcclient/examples/dcrwalletwebsockets)  
-  Connects to a dcrwallet RPC server using TLS-secured websockets, registers for
-  notifications about changes to account balances, and gets a list of unspent
-  transaction outputs (utxos) the wallet can sign
 
 ## Major Features
 
 * Supports Websockets (dcrd/dcrwallet) and HTTP POST mode (bitcoin core-like)
-* Provides callback and registration functions for dcrd/dcrwallet notifications
+* Provides callback and registration functions for dcrd notifications
 * Supports dcrd extensions
```

Comment:
This was already there, but it's incorrect and should probably be updated.  It's a holdover from referring to btcd extensions versus BC.  There are no dcrd extensions.